core/broadleaf-framework-web/src/main/java/org/broadleafcommerce/core/web/controller/account/BroadleafManageWishlistController.java

```diff
@@ -48,7 +49,7 @@
     protected static String accountWishlistRedirect = "redirect:/account/wishlist";
 
     public String add(HttpServletRequest request, HttpServletResponse response, Model model,
-            AddToCartItem itemRequest, String wishlistName) throws IOException, AddToCartException, PricingException  {
+                      ConfigurableOrderItemRequest itemRequest, String wishlistName) throws IOException, AddToCartException, PricingException  {
```

Comment:
This needs to be OrderItemRequestDTO instead. First of all for compatibility reasons (i.e. clients may be using this method and sending it AddToCartItem which subclasses OrderItemRequestDTO however it doesn't subclass ConfiguratbleOrderItemRequest) and second of all if we only need it to be an OrderItemRequestDTO then there's not reason to require a more specific implementation